src/02-common-principles.md

```diff
@@ -161,6 +161,28 @@ A summary of all entities in BIDS and the order in which they MUST be
 specified is available in the [entity table](./99-appendices/04-entity-table.md)
 in the appendix.
 
+### Entity-linked file collections
+
+An entity-linked file collection is a set of files that are related to each other
+based on a repetitive acquisition of sequential data
+by changing acquisition parameters one (or multiple) at a time
+or by being inherent components of the same data.
+Entity-linked collections are identified by a common suffix,
+indicating the group of files that should be considered a logical unit.
+Within each collection, files MUST be distinguished from each other by at least one 
+entity (for example, `echo`) that corresponds to an altered acquisition parameter
+(`EchoTime`) or that defines a component relationship (for example, `part`).
+Note that these entities MUST be described by the specification and
+the parameter changes they declare MUST NOT invalidate the definition of the accompanying suffix.
+For example, the use of the `echo` entity along with the `T1w` suffix casts doubt on
+the validity of the identified contrast weighting.  
+Provided the conditions above are satisfied,
+any suffix (such as `bold`) can identify an entity-linked file collection,
+although certain suffixes are exclusive for this purpose (for example, `MP2RAGE`).
+Use cases concerning this convention are compiled in the
+[file collections](./99-appendices/10-file-collections.md) appendix.
+This convention is mainly intended for but not limited to MRI modalities.
+
```

Comment:
Could a practical example be added so it's easier to understand?

Reply:
cc @agahkarakuzu

Reply:
@guiomar we have example datasets at https://osf.io/k4bs5/. However, we need to update those file names to account for the the latest changes. We have a few PRs left to complete BEP001, which will make this clearer.